src/controllers/authenticationController.ts

```diff
@@ -224,6 +233,40 @@ export default class AuthenticationController {
 
     return ResponsePermanentRedirect(urlWithToken);
   }
+
+  public async acsTest(
+    userPayload: unknown
+  ): Promise<
+    // tslint:disable-next-line: max-union-size
+    | IResponseErrorInternal
+    | IResponseErrorValidation
+    | IResponseErrorForbiddenNotAuthorized
+    | IResponseSuccessJson<AccessToken>
+  > {
+    const acsResponse = await this.acs(userPayload);
+    if (acsResponse.kind === "IResponsePermanentRedirect") {
+      const REDIRECT_URL = clientProfileRedirectionUrl.replace("{token}", "");
```

Comment:
may we add a comment on what this code does and why?

Reply:
fixed in ec700e0